os/dbuild.sh

```diff
@@ -26,7 +26,7 @@ BINDIR="${BUILDDIR}/output/bin"
 CONFIGDIR="${BUILDDIR}/configs"
 DOCKER_VERSION="1.5.5"
 
-STATUS_LIST="NOT_CONFIGURED BOARD_CONFIGURED CONFIGURED BUILT PREPARE_DL DOWNLOAD"
+STATUS_LIST="NOT_CONFIGURED BOARD_CONFIGURED CONFIGURED BUILT PREPARE_DL DOWNLOAD_READY"
```

Comment:
Can you pls check all the other places in this file where "`DOWNLOAD`" is used and change it accordingly?

Reply:
Checked and updated